docs/extending/extending_draftail.md

```diff
@@ -2,15 +2,17 @@
 
 # Extending the Draftail Editor
 
-Wagtail’s rich text editor is built with [Draftail](https://www.draftail.org/), and its functionality can be extended through plugins.
+Wagtail’s rich text editor is built with [Draftail](https://www.draftail.org/), which supports different types of extensions.
 
-Plugins come in three types:
+## Formatting extensions
```

Comment:
Changes in this first section aren’t super related to the new customisations. I chose to make them anyway because I mention compatibility between extensions further down – so feels relevant to mention what is compatible or not here too.